UIforETW/Support.cpp

```diff
@@ -167,8 +167,8 @@ int CUIforETWDlg::BufferCountBoost(int requestCount) const
 	if (!GlobalMemoryStatusEx(&memoryStatus))
 		return requestCount;
 
-	const int64_t oneGB = int64_t(1024) * 1024 * 1024;
-	int64_t physicalRam = memoryStatus.ullTotalPhys;
+	const int64_t oneGB = 1024i64 * 1024i64 * 1024i64;

```

Comment:
1024i64 is almost unreadable I think. At a glance it reads like 1024164 which is a very strange constant.